templates/test/dev/cluster-template-custom-builds-load.yaml

```diff
@@ -261,14 +420,20 @@ spec:
         windowsServerVersion: ${WINDOWS_SERVER_VERSION:=""}
     spec:
       identity: UserAssigned
+      image:
+        marketplace:
```

Comment:
And for Windows images in the Community Gallery:

```yaml
image:
  computeGallery:
    gallery: "ClusterAPI-f72ceb4f-5159-4c26-a0fe-2ea738f0d019"
    name: "capi-win-2022-containerd"
    version: "latest"
```

windows-2019 is now deprecated and we won't be publishing it any more: see #5357.